orchestration/prepare.go

```diff
@@ -96,3 +119,68 @@ func PrepareOrchestrator(ctx context.Context, targetEnv cfg.Configuration, clust
 
 	return &RemoteOrchestrator{cluster: boundCluster, server: focus.Proto(), endpoint: endpoint}, nil
 }
+
+func requiresUpdate(ctx context.Context, cluster runtime.Cluster, server planning.Server, prebuilts []*schema.Workspace_BinaryDigest) (bool, error) {
```

Comment:
Since this method requires a planning.Server, we need to spend cycles doing package loading. Ideally this check runs before doing any package loading. You only need the server to get a proto, so you get the id. I think we can either (a) hardcode the id, or (b) have a separate label that is hardcoded.